src/cpp/fastdds/builtin/typelookupservice/TypeLookupManager.hpp

```diff
@@ -24,9 +24,9 @@
 #include <vector>
 #include <mutex>
 
-#include <fastdds/dds/builtin/typelookup/TypeLookupRequestListener.hpp>
-#include <fastdds/dds/builtin/typelookup/TypeLookupReplyListener.hpp>
-#include <fastdds/dds/builtin/typelookup/common/TypeLookupTypes.hpp>
+#include <fastdds/builtin/typelookupservice/TypeLookupRequestListener.hpp>
+#include <fastdds/builtin/typelookupservice/TypeLookupReplyListener.hpp>
+#include <fastdds/builtin/typelookupservice/common/TypeLookupTypes.hpp>
 
 #include <fastdds/rtps/builtin/data/WriterProxyData.h>
 #include <fastdds/rtps/builtin/data/ReaderProxyData.h>
```

Comment:
Please, follow the recommended including order. As these headers are private now, they should be included the latest, sorted alphabetically with the other internal header files. Please, apply this suggestion to the other files.

Reply:
Fixed in [2bfaa40](https://github.com/eProsima/Fast-DDS/pull/4053/commits/2bfaa40548d3668e183c3d0a84e25647ea09675e)